plugins/flattener/profile.json

```diff
@@ -4,9 +4,9 @@
     "events": [],
     "methods": [],
     "version": "0.1.0",
-    "url": "https://remix-flattener.netlify.com",
+    "url": "https://talented-holiday.surge.sh",
```

Comment:
Change `https://remix-flattener.netlify.com` to `https://remix-flattener.netlify.app` should make the trick.